packages/hoppscotch-common/src/components/smart/EnvInput.vue

```diff
@@ -109,6 +114,7 @@ const props = withDefaults(
     contextMenuEnabled?: boolean
     secret?: boolean
     autoCompleteEnv?: boolean
+    placeholderHoverString: string
```

Comment:
There are many instances where `EnvInput` is used without a `placeholderHoverString`. So this should not be a required prop.